aws-route53/main.tf

```diff
@@ -23,6 +23,11 @@ variable "ssh_key" {
   default     = ""
 }
 
+variable "zone" {
```

Comment:
is this rebased? this variable is already on master

Reply:
yes I reverted by previous change and rebased.

Reply:
so why is this variable change still in the PR?